apps/builder/app/builder/shared/meta-icon.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = {
+  size?: number | string;
+  icon: string;
+};
+
+export const MetaIcon = ({ size = 16, icon }: Props) => {
+  return (
+    <div
+      style={{ width: size, height: size }}
```

Comment:
I would not use inline style, since you can have reusable css

Reply:
Would probably use a span, but no strong opiinion

Reply:
check out https://www.radix-ui.com/docs/primitives/utilities/accessible-icon

Reply:
depending on how icon is used, you may need to announce it accessibly and/or hide it from screen readers check out what radix does with icons